portfolio/src/main/java/com/google/sps/data/CommentHistory.java

```diff
@@ -0,0 +1,74 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class representing the subtraction game, where players take turns subtracting from 21 to reach 0.
```

Comment:
This comment is confusing to me - this class doesn't have anything to do with the subtraction game, does it?

I'm assuming this was just copied over by accident from an example file, make sure to update it with a brief description of this class. Maybe just "Represents a comment on my website" or something.